Formula/yash.rb

```diff
@@ -18,21 +18,16 @@ class Yash < Formula
     sha256 mojave:        "0eec194f969eb03add9615ed1f4569749ff423007552fdf40b12b7df82c72024"
   end
 
+  head do
+    url "https://github.com/magicant/yash.git", branch: "trunk"
+
+    depends_on "asciidoc" => :build
+  end
+
   depends_on "gettext"
 
   def install
-    system "sh", "./configure",
-            "--prefix=#{prefix}",
-            "--enable-array",
-            "--enable-dirstack",
-            "--enable-help",
-            "--enable-history",
-            "--enable-lineedit",
-            "--enable-nls",
-            "--enable-printf",
-            "--enable-socket",
-            "--enable-test",
-            "--enable-ulimit"
+    system "sh", "./configure", "--prefix=#{prefix}"
```

Comment:
I actually would add `--disable-test` and would use `std_configure_args`. It'll be something like this:

```suggestion
    args = std_configure_args + %w[
      --disable-test
    ]
    system "sh", "./configure", *args
```

Reply:
We could just pass that straight to `configure` since there's only one additional flag to `std_configure_args`.

Reply:
> I actually would add `--disable-test`

Note that this would disable the `test` shell builtin. It's not for disabling test running or anything like that.

https://github.com/magicant/yash/blob/b84449beea3d645f24fc3356761efc9363fd739c/INSTALL#L108-L110

> use `std_configure_args`

The `configure` script for yash is hand written, not from autoconf. I don't think the standard args would work

Reply:
Riiight... I'm ok with initially suggested changes then, thanks @algorythmic!